pkg/render/node.go

```diff
@@ -384,6 +384,14 @@ func (c *nodeComponent) nodeRole() *rbacv1.ClusterRole {
 				},
 				Verbs: []string{"create"},
 			},
+			{
+				// Tigera Secure updates status for packet captures.
+				APIGroups: []string{"crd.projectcalico.org"},
+				Resources: []string{
+					"packetcaptures",
```

Comment:
Would `packetcaptures/status` work (like `pod/status` at Line 281)?

Reply:
Not really. I tried using only packetcapture/status, but it did not work.

Reply:
As you pointed out in slack, libcalico-go doesn't provide a Status only update mechanism so I think that is why this isn't possible. Bummer.